app/presenters/task_list_presenter.rb

```diff
@@ -1,5 +1,5 @@
 class TaskListPresenter
-  TaskListRow = Struct.new(:title, :minutes_to_complete, :value, :start_path)
+  TaskListRow = Struct.new(:title, :minutes_to_complete, :value, :start_path, :can_start?)
```

Comment:
As discussed - Let's change this to an OpenStruct, in a future version. 5 positional parameters is icky.